sdk/openai/Azure.AI.OpenAI/src/Custom/AzureOpenAIClient.cs

```diff
@@ -342,7 +342,7 @@ private static PipelinePolicy CreateAddUserAgentHeaderPolicy(AzureOpenAIClientOp
     private static readonly string s_aoaiEndpointEnvironmentVariable = "AZURE_OPENAI_ENDPOINT";
     private static readonly string s_aoaiApiKeyEnvironmentVariable = "AZURE_OPENAI_API_KEY";
     private static readonly string s_userAgentHeaderKey = "User-Agent";
-    private static PipelineMessageClassifier _pipelineMessageClassifier200;
-    internal static PipelineMessageClassifier PipelineMessageClassifier200
-        => _pipelineMessageClassifier200 ??= PipelineMessageClassifier.Create(stackalloc ushort[] { 200 });
+
+    internal static PipelineMessageClassifier PipelineMessageClassifier { get; } =
```

Comment:
Why merge the lazy init? It's pragmatically not a huge issue, but I was asked to keep the `??=` property/field pattern after having it pointed out that this imposes a frontloaded allocation even if you never use it (e.g. if you just preemptively included the Azure library but didn't call to Azure yet)